src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -80,8 +79,6 @@
 
         function initialize(requestVerificationToken, returnUrl, loginOptions, personalIdentityNumber, personalIdentityNumberElement) {
             formFieldsetElement.disabled = true;
-            show(formSubmitButtonSpinnerElement);
```

Comment:
As stated in the general comment, I don't think we should remove the spinner from the button. If we place the image in css (using css background), we can still reuse it.

Placing it in CSS makes it harder to change color though, hmm, if we simplify the generated SVG, extracts the animation into CSS and have the SVG itself inlined we can control the color of the icon using CSS as well.